data/json/monsters/zed_fusion.json

```diff
@@ -211,6 +211,92 @@
     "death_function": [ "GAS" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "POISON", "CLIMBS", "NO_BREATHE", "CLIMBS", "HARDTOSHOOT" ]
   },
+  {
+    "id": "mon_zombie_scissorlimbs",
+    "type": "MONSTER",
+    "name": { "str": "scissorlimbs" },
+    "description": " A nightmarish spider of gore stands tall among the ruins, and keeps silent watch of the blighted landscape.  Its spindly limbs of bone slip between the rubble with otherworldly speed.",
+    "default_faction": "zombie",
+    "bodytype": "spider",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 5,
+    "volume": "122500 ml",
+    "weight": "81500 g",
+    "hp": 10,
+    "speed": 250,
+    "material": [ "flesh" ],
+    "symbol": "M",
+    "color": "red_yellow",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 5,
+    "melee_dice_sides": 2,
+    "melee_cut": 12,
+    "armor_bash": 2,
+    "armor_cut": 10,
+    "armor_bullet": 10,
+    "vision_day": 1,
+    "special_attacks": [ { "type": "leap", "cooldown": 5, "max_range": 3, "allow_no_target": true }, [ "scratch", 5 ] ],
+    "death_function": [ "GAS" ],
+    "flags": [ "SEES", "HEARS", "GOODHEARING", "WARM", "POISON", "CLIMBS", "NO_BREATHE", "CLIMBS", "HARDTOSHOOT" ]
```

Comment:
You say it's completely blind yet it has the SEES flag?

Reply:
Only one tile to ensure it can hit you when adjacent.
Which is the bare minimum for hostile monsters, I think?